gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,99 @@
+//tag::intro1[]
+
+Flex Gateway offers the following CLI command to test the health of a Flex Gateway:
+
+[source,ssh]
+----
+flexctl probe --check=liveness
+----
+
+This command returns either a status code of `0` for a healthy Flex Gateway or `1` for an unhealthy Flex Gateway.
+
+//end::intro1[]
+
+//tag::intro2[]
+By using the liveness check command, you can either configure a Kubernetes liveness probe or a Docker health check command to restart a failed Flex Gateway.
+
+For Flex Gateways running on Linux, you can still run the liveness check command. However, there is not an easy to automate the Linux machine to restart when the command returns an unhealthy status.
+
+//tag::intro2[]
+
+//tag::k8s-liveness-probe[]
+== Configure a Kubernetes Liveness Probe
+
+For Flex Gateways running on Kubernetes, you can configure a Kubernetes liveness probe to monitor and restart a Flex Gateway pod when unhealthy. To learn more about Kubernetes probes, see https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/[Configure Liveness, Readiness and Startup Probes^].
+
+To enable a liveness probe, update you Helm chart with the following configuration:
```

Comment:
Here we need to mention that users can customize the probe, but that one is provided by default